### PR TITLE
Update windows_git_for_windows_turn_on_ssh_agent.md

### LINKED ISSUE #29644: Shell code formatting is incorrect
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/pt/authentication/troubleshooting-ssh/error-permission-denied-publickey#make-sure-you-have-a-key-that-is-being-used

### What part(s) of the article would you like to see updated?

Shell code formatting is incorrect:

image

This incorrect formatting can make it difficult to read the documentation.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by igorjcl**
### Why: Shell code formatting is incorrect

Closes: https://github.com/github/docs/issues/29644

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Review by subatoi — DISMISSED** [review]
Thank you! This is the correct fix: I actually had a PR open internally but I'm very happy to accept this one instead :)

File: data/reusables/desktop/windows_git_for_windows_turn_on_ssh_agent.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
-    ```shell
-    # start the ssh-agent in the background
-    $ eval $(ssh-agent -s)
-    > Agent pid 59566
-    ```
+```shell
+  # start the ssh-agent in the background
+  $ eval $(ssh-agent -s)
+  > Agent pid 59566
+```
```
Comment (subatoi) [inline data/reusables/desktop/windows_git_for_windows_turn_on_ssh_agent.md:7]:
```suggestion
```shell
# start the ssh-agent in the background
$ eval $(ssh-agent -s)
> Agent pid 59566
```
```

Ah, just need to adjust indentation here then the build should pass 🤞

File: data/reusables/desktop/windows_git_for_windows_turn_on_ssh_agent.md [hunk 2]
```diff
@@ -1,7 +1,6 @@
...
 
-    ```shell
-    # start the ssh-agent in the background
-    $ eval $(ssh-agent -s)
-    > Agent pid 59566
-    ```
+```shell
+# start the ssh-agent in the background
+$ eval $(ssh-agent -s)
+> Agent pid 59566
```
Comment (subatoi) [inline data/reusables/desktop/windows_git_for_windows_turn_on_ssh_agent.md:6]:
```suggestion
> Agent pid 59566
```
```

Comment (subatoi) [inline data/reusables/desktop/windows_git_for_windows_turn_on_ssh_agent.md:6]:
```suggestion
```shell
# start the ssh-agent in the background
$ eval $(ssh-agent -s)
> Agent pid 59566
```

```

**Comment by subatoi** [comment]
Apologies! My suggestion to fix the indentation managed to remove the terminating backticks, and I don't have permission to push to the branch to rectify it. I'm happy to approve this PR when the build succeeds: as far as I can tell, the reason for the failure is probably transient. If adding the backticks and updating the branch doesn't fix the build, I'll follow up with the engineering team internally. 

Thanks again for raising this and sorry for the confusion: do please let me know if you'd prefer me to get this change merged internally and I'm happy to add you as co-committer 😄

**Comment by igorjcl** [comment]
@subatoi, thank you for your attention. I prefer the change to be made internally. Please proceed with the necessary alterations, and I would appreciate it if you could add me as a co-author.

**Comment by subatoi** [comment]
No problem @igorjcl — thanks for your patience!

I've just committed a fix for this. You should no longer see the issue you reported on https://docs.github.com/pt/authentication/troubleshooting-ssh/error-permission-denied-publickey#make-sure-you-have-a-key-that-is-being-used. You may need to open an incognito browser or send a junk query string to see it on the live page, if you don't see it straightaway. 

When the next repo-sync runs, you should see yourself as co-committer. If you're happy with all this, we can close this issue out!

**Comment by subatoi** [comment]
Just to add: this is the commit I was referring to https://github.com/github/docs/commit/c512459f9875a822811d598483e83722431c57b5

I can see the changes haven't propagated yet to the Portuguese language version. I'll check back with this tomorrow but I can see the example is fixed on the English language version.

**Comment by subatoi** [comment]
Just to let you know I've followed up internally: the Portuguese version should see the updates soon. I'll check back next week!

**Comment by Bo-Bo-Han** [comment]
```

> ```ass```

# ```~~~~

**Comment by igorjcl** [comment]
Thanks @subatoi

**Comment by subatoi** [comment]
Hi @igorjcl I just wanted to follow up again: the fix has been shipped to the non-English docs now (including Portuguese). Thanks again for raising this and apologies that it took so long to appear there!

**Pull request closed without merging**